vroom/vim.py

```diff
@@ -120,6 +124,13 @@ def Start(self):
     # still has a _vim attribute it can query for details.
     self.process = subprocess.Popen(self.start_command, env=self.env)
     time.sleep(self.args.startuptime)
+    if self.process.poll() is not None and self.process.poll() != 0:
+      # If vim exited this quickly, it probably means we passed a switch it
+      # doesn't recognize. Try again without the '--clean' switch since this is
+      # new in 8.0.1554+.
+      self.start_command.remove('--clean')
```

Comment:
FYI, I'm thinking of eventually plumbing through and post a warning notice in the final output if we couldn't use `--clean`.